src/app/components/challenge/challengesubmissions/challengesubmissions.component.html

```diff
@@ -69,102 +69,92 @@
       </div>
```

Comment:
We already have a check mark, "Shown on leaderboard" in each collapsible. So is this needed now?

Reply:
Removed that check once.